src/components/Radio/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { nanoid } from 'nanoid'
+import React, { InputHTMLAttributes } from 'react'
+
+import { ColorTheme } from '../../enum/ColorTheme'
+import {
+	ComponentSize,
+	ComponentSizeMediumLarge
+} from '../../enum/ComponentSize'
+import { StyledRadio } from './RadioStyles'
+
+export interface RadioProps
+	extends Omit<InputHTMLAttributes<HTMLInputElement>, 'size'> {
+	rtl?: boolean
+	colorTheme?: ColorTheme
+	size?: ComponentSizeMediumLarge
+}
+
+const Radio: React.FC<RadioProps> = ({
+	colorTheme = ColorTheme.PRIMARY,
+	size = ComponentSize.MEDIUM,
+	className,
+	rtl,
+	children,
+	...props
+}) => {
+	const id = props.id || nanoid()
```

Comment:
Let's forget about `||` 🙂 
```suggestion
	const id = props.id ?? nanoid()
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
OK, I'll fix it.